manual/python/index.md

```diff
@@ -87,9 +173,53 @@ root [2] TPython::Prompt()
 
 For a more complete description of the `TPython` interface, please check the reference guide for {% include ref class="TPython" %}.
 
-## JupyROOT: PyROOT for Jupyter notebooks
+## JupyROOT: (Py)ROOT for Jupyter notebooks
+
+[Jupyter notebooks](https://jupyter.org/) provide a web-based interface that can be used to do interactive computing with ROOT.
+
+### How to use it
+
+Two kernels (or language flavours) allow to run ROOT from Jupyter: *Python* (2 and 3) and *C++*. In order to use such kernels, there are mainly two options:
+
+- Local ROOT installation: to install ROOT locally, please follow the instructions [here]({{ '/install' | relative_url }}). In addition to ROOT, two Python packages are required, which can be installed e.g. with `pip`:
+
+```shell
+pip install jupyter
+pip install metakernel
+```
+
+Once all requirements are installed, a Jupyter server with the Python and C++ kernels can be started by running from a terminal:
+
+```shell
+root --notebook
+```
+
+- SWAN: CERN provides an online service to create and run Jupyter notebooks, called [SWAN](https://swan.cern.ch). With this option no installation is required: a browser and a working Internet connection are enough. Both the Python and C++ kernels are available in SWAN.
```

Comment:
`internet`, lower case

Reply:
But Internet is a proper noun, I think uppercase is better.